PyDMXControl/controllers/_serialController.py

```diff
@@ -0,0 +1,37 @@
+"""
+ *  PyDMXControl: A Python 3 module to control DMX using uDMX.
+ *                Featuring fixture profiles, built-in effects and a web control panel.
+ *  <https://github.com/MattIPv4/PyDMXControl/>
+ *  Copyright (C) 2018 Matt Cowley (MattIPv4) (me@mattcowley.co.uk)
+"""
+
+import serial as sr
```

Comment:
I personally don't see any reason to alias serial to sr, to me it makes more sense to just use serial to avoid any confusion?

Reply:
I'm using direct imports now.